### DISCUSSION TITLE
Spare PCB Boards for Germany

### DISCUSSION THREAD

**Discussion by mbeetle** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#305] [General]
Hi
I have 3 V1 boards left for shipping in Germany. I didn't want to wait long so I ordered it by express. That's why I'm selling them for €3 + €1 postage.

This is the Board:
![BoardVV1](https://user-images.githubusercontent.com/66788439/175367712-a03294cc-dc03-4ce0-b469-16e6281f5b51.png)

**Comment by eht584** [comment]
Hallo, ich werde kaufen, PayPal ?
Grüß aus Siegburg
Eduard

**Comment by mbeetle** [comment]
Hallo Eduard

Gerne per Paypal an ***@***.***
Schick mir dann deine Adresse.

Viele Grüße
Michael


eht584 ***@***.***> schrieb am Fr., 24. Juni 2022, 08:50:

**Comment by eht584** [comment]
Die E-Mail Adresse wird ausgeblendet, schick mit bitte was an e.herdt()freenet.de

**Comment by BIIINO** [comment]
Hi i am interested. Are you willing to send it also in Slovakia please ?

feel free to contact me: bino@imafex.sk

**Reply by mbeetle** [reply]
Hi. What kind of shipping would you like? by regular mail?

**Comment by BIIINO** [comment]
Hello, thank tou for ur answer. I dont care, what type, i want is as soon as possible. Of course payment befofe shipping

Odoslané z iPhonu
